docs/cloud/rahti/security-guide.md

```diff
@@ -1,27 +1,26 @@
-# Security guide
-
-Rahti applications are exposed to the Internet, and
+--8<-- "rahtibeta_announcement.md"
+Rahti 1 applications are exposed to the Internet, and
 their security should be treated with an appropriate care.
-The user on whose account a service is running in Rahti is
+The user on whose account a service is running in Rahti 1 is
 responsible for its security. See the [terms of use](https://rahti.csc.fi/agreements/terms_of_use/) for details about the
```

Comment:
Rahti 2 has new terms of use: https://wiki.csc.fi/ContainerPlatforms/RahtiServiceTermsofUse

Reply:
I guess that the link in the documentation will still be valid, as you are going to update the terms of use page? Or will you publish these new terms of use in a new public link?

Reply:
The content of https://rahti.csc.fi will be replaced by the content of https://landing.2.rahti.csc.fi/. The problem now is that  https://landing.2.rahti.csc.fi/ has no section for terms of use, and instead refers to the old terms of use (under Service description). So yes we need to host the new terms of user somewhere else. Is it fine to add a page for them in Rahti documentation (probably the easiest solution)?

Reply:
IMHO, feel free to publish the terms anywhere. 

I think it is easier that you publish it in <https://landing.2.rahti.csc.fi/agreements/terms_of_use/> that will be later published in the old link. But as it is you who wrote them, you decide.